Library/Formula/filebeat.rb

```diff
@@ -0,0 +1,31 @@
+class Filebeat < Formula
+  desc "Log data shipper based on Logstash-Forwarder"
+  homepage "https://www.elastic.co/guide/en/beats/filebeat/current/index.html"
+  url "https://github.com/elastic/beats/archive/v1.1.1.tar.gz"
+  sha256 "5cf4751e6cd5199e6cef5b9d86ef62923901e659f4f15c70a4a9e007c32c4eb5"
+
+  depends_on "go"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/elastic/"
+    ln_sf buildpath, buildpath/"src/github.com/elastic/beats"
+
+    cd "src/github.com/elastic/beats/filebeat" do
+      system "make", "build"
+      libexec.install "filebeat"
+      prefix.install "etc/filebeat.template.json"
+      etc.install "etc/filebeat.yml"
+    end
+
+    (bin/"filebeat").write <<-EOS.undent
+    #!/usr/bin/env bash
+    #{libexec}/filebeat -c #{etc}/filebeat.yml $@
+    EOS
+    chmod 0755, bin/"filebeat"
+  end
+
+  test do
+    system bin/"filebeat", "-version"
```

Comment:
Ok I checked, and sadly I don't see any way to do a better test.
For starter it's no easy to test that the behaviour expected from filebeat is OK, only way I'd think you be to start filebeat with a test configuration pointing to an empty test log file and output in a file, then write something in the test log file and test that the output file has new data.
Only problem is that there is not way (that I know) to start filebeat as a daemon, and trying to `spawn` it or stuff like that won't do because actually 2 filebeat processes are spawned, so getting a PID to kill is not enough :frowning: 
I don't really have a solution here, but I'm open to any suggestion.